Source/Xamarin/Prism.Forms/Navigation/PageNavigationService.cs

```diff
@@ -375,6 +376,18 @@ protected virtual Page CreatePageFromSegment(string segment)
             }
         }
 
+        protected void ApplyPageBehaviors(Page page)
```

Comment:
I think we can keep it simple for now. I'm not sure how common it is to apply a behavior to every instance of a specific page in an app.